README.md

```diff
@@ -161,7 +161,45 @@ You can force the direction of a relationship by using:
 * `Rel_L`, `Rel_Left`
 * `Rel_R`, `Rel_Right`
 
-In rare cases, you can force the layout using hidden relationships, `Lay_U`, `Lay_D`, `Lay_L`, `Lay_R`.
+In rare cases, you can force a layout of objects which has no relationship by using:
```

Comment:
In rare cases, you can force the layout of objects which have no relationships by using:

Not a native speaker but I think it's grammatically correct